tests/unit/models/cart-model-test.js

```diff
@@ -73,20 +73,57 @@ test('it proxies sub total from the underlying cart', function (assert) {
   assert.equal(model.subtotal, cartFixture.cart.subtotal_price);
 });
 
-test('it creates a line item when you add a variant', function (assert) {
+test('it deprecates addVariants to createLineItemsFromVariants', function (assert) {
```

Comment:
How would you go about testing that `addVariants` calls `createLineItemsFromVariants`?

My thought was for the end user it doesn't really matter as long as the functionality is the same.